src/nih_wayfinding/app/index.html

```diff
@@ -28,7 +28,10 @@
         </div>
 
         <div class="navbar-header pull-right">
-          <div class="btn-group" dropdown>
+          <button ng-if="navbar.config.back" type="button" class="btn btn-primary" ng-click="navbar.back()">
```

Comment:
Is this what we have in the wireframes?  Seems like back buttons usually go on the left, so putting on the right might be confusing.